### PR TITLE
zh-translation: /docs/ops/best-practices/security/index.md #1732

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1732](https://github.com/servicemesher/istio-official-translation/issues/1732)

### CODE REVIEW

File: content/zh/docs/ops/best-practices/security/index.md [hunk 1]
```diff
@@ -1,43 +1,23 @@
...
-For example, you can create a `team1-ns` namespace for `team1`, and `team2-ns` namespace for `team2`, such
-that both teams cannot access each other's services.
+如果有多个运维人员（也称为 [SRE](https://en.wikipedia.org/wiki/Site_reliability_engineering)）在大型或中型集群中部署不同的服务，我们建议为每个 SRE 团队创建一个单独的 [Kubernetes 命名空间](https://kubernetes.io/docs/tasks/administer-cluster/namespaces-walkthrough/)，以隔离他们的访问权限。
+例如，您可以为 `team1` 创建一个 `team1-ns` 命名空间，为 `team2` 创建一个 `team2-ns` 命名空间，以使两个团队无法访问彼此的服务。
 
 {{< warning >}}
-If Citadel is compromised, all its managed keys and certificates in the cluster may be exposed.
-We **strongly** recommend running Citadel in a dedicated namespace (for example, `istio-citadel-ns`), to restrict access to
-the cluster to only administrators.
+如果 Citadel 受到威胁，则它在群集中的所有托管密钥和证书都可能泄露。我们强烈建议您在专用命名空间（例如，`istio-citadel-ns` ）中运行 Citadel，以将对群集的访问限制为，仅管理员可访问。
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/best-practices/security/index.md]:
```suggestion
如果 Citadel 受到威胁，则它在集群中的所有托管密钥和证书都可能泄露。我们强烈建议您在专用命名空间（例如，`istio-citadel-ns` ）中运行 Citadel，来限制集群的访问权限，仅适用于管理员。
```

Comment (gorda) [reply content/zh/docs/ops/best-practices/security/index.md]: @yuxiaobo96  感谢细心 review，已修改

**Pull request merged**